db/system_distributed_keyspace.cc

```diff
@@ -151,4 +186,99 @@ future<> system_distributed_keyspace::remove_view(sstring ks_name, sstring view_
             false).discard_result();
 }
 
+/* When managing CDC descriptions, we will require CDL = ONE reads/writes
+ * only if there's only one running node. Otherwise, we'll require CL = QUORUM.
+ */
+static db::consistency_level quorum_if_many() {
+    auto& gossiper = gms::get_local_gossiper();
```

Comment:
Global call detected! We're removing those, not adding more.

Reply:
Done